docker-stage1/Dockerfile

```diff
@@ -6,8 +6,8 @@ ADD scripts /scripts
 
```

Comment:
Do we need to update the base image to `ubuntu:20.10` or `debian:bullseye`?

Reply:
I would highly advice to update to `debian` yes.